app/Http/Requests/Sharing/DeleteSharingRequest.php

```diff
@@ -22,7 +22,7 @@ class DeleteSharingRequest extends BaseApiRequest
 	 */
 	public function authorize(): bool
 	{
-		return Gate::check(UserPolicy::CAN_UPLOAD, User::class);
+		return Gate::check(UserPolicy::MAY_UPLOAD, User::class);
```

Comment:
Again, the result of a policy has to be a right, in this case I would use the same as for adding shares, i.e. `Album::CAN_SHARE`.